src/configure/configurers/azurePipelineConfigurer.ts

```diff
@@ -96,6 +96,7 @@ export class AzurePipelineConfigurer implements Configurer {
                             id: repository.project.id,
                             name: repository.project.name
                         };
+                        telemetryHelper.setTelemetry(TelemetryKeys.RepoId, inputs.sourceRepository.repositoryId);
                     });
```

Comment:
Wasn't repoId already getting stored? Or this is some other scenario which wasn't covered before?

Reply:
For the ADO repo, the id was null where we were setting it earlier in the telemetry.
Here only we get the ID of the repo.
Similarly for Local repo scenario, the ID were null at the time of setting up.